docs/rules/index.md

```diff
@@ -5,42 +5,45 @@ layout: doc
 <!-- Note: No pull requests accepted for this file. See README.md in the root directory for details. -->
```

Comment:
We can definitely try it. Problem with local jekyll vs. github is that they are not exactly the same. But we can try and see if it works.